### PR TITLE
My trial project

### PR DESCRIPTION
**Pull request by DavidCazco36**
Survey form

### CODE REVIEW

**Review by DavidCazco36 — COMMENTED** [review]
I suggest you not to modify the comment part.

**Review by DavidCazco36 — COMMENTED** [review]
Kindly, I suggest You to added a more profesional readme file. You could use [this template](https://github.com/microverseinc/readme-template/blob/master/README.md)

File: index.html [hunk 1]
```diff
@@ -45,7 +45,7 @@ <h1 id="title">Survay Form</h1>
     Feel free to delete this message once you have read it.
     Good luck and Happy Coding!
 
-    - The freeCodeCamp Team
+    - The freeCodeCamp Team :)
```
Comment (DavidCazco36) [inline index.html:48]: -I suggest you to make more descriptive commits.
-Kindly, you should not modify the comment section.

Comment (bienvenuushindi) [reply index.html:48]: - [ ] :red_circle: [REQUIRED] This comment is unclear. Please make sure when you comment you also give more details about it. By this I mean, tell the project's owner why we need a more descriptive commit and how a descriptive commit looks like. You could say for example
 **I suggest you make a more descriptive commit message. Commit messages should be meaningful and should tell what changes are inside. For example, if you add a title to your index.html the respective commit message can be something  like  `Add page title  to index.html `**

Comment (bienvenuushindi) [reply index.html:48]: - [x] :red_circle: [REQUIRED] This comment is unclear. Please make sure when you comment you also give more details about it. By this I mean, tell the project's owner why we need a more descriptive commit and how a descriptive commit looks like. You could say for example
 **I suggest you make a more descriptive commit message. Commit messages should be meaningful and should tell what changes are inside. For example, if you add a title to your index.html the respective commit message can be something  like  `Add page title  to index.html `**

**Review by bienvenuushindi — CHANGES_REQUESTED** [review]
# Changes Required ♻️
### Hi :wave: @DavidCazco36  ,

Nice job :clap: :100: , you are almost there.



## (Highlights) Good Points: :+1:

- :heavy_check_mark: Descriptive PR title.
- :heavy_check_mark: Inline comment is clear and easy to understand.

## Required Changes :recycle: 

 - [x] Your review should have at least two comments. So far you have just one comment. Please add one more comment as requested in the [project requirements ](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/review_your_project.md#project-requirements)
 > requirement

- [x] Kindly check comments under the review.

## [Optional] suggestions:

- [ ] I think it will be great if you use a couple of emojis and/or gifs in the review.They make the code review more human-friendly 👽🤖.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html (hunk 1 above)
Comment (DavidCazco36) [reply index.html:48]: - [x] I suggest you make a more descriptive commit message. [This article could help you with that.](https://www.freecodecamp.org/news/how-to-write-better-git-commit-messages/) Commit messages should be meaningful and should tell what changes are inside. For example, if you add a title to your index.html the respective commit message can be something like Add page title to index.html

Comment (DavidCazco36) [inline index.html:48]: - [x] I suggest you to add a profesional ReadMe file. You can use [this template.](https://github.com/microverseinc/readme-template) Remember that a ReadMe file should contain important information about your project like (description about it, how to set up the project, authors and aditional indications if needed).

**Review by binhussen — APPROVED** [review]
Hi @DavidCazco36 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

## Highlights
## Optional suggestions
- good job 👍 I think it is better to create your PR with some additional  changes,  because  this activity is for practice.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**